sample/src/main/java/org/hawkular/accounts/sample/boundary/SampleEndpoint.java

```diff
@@ -16,16 +16,7 @@
  */
 package org.hawkular.accounts.sample.boundary;
 
-import org.hawkular.accounts.api.PermissionChecker;
-import org.hawkular.accounts.api.ResourceService;
-import org.hawkular.accounts.api.NamedOperation;
-import org.hawkular.accounts.api.model.Operation;
-import org.hawkular.accounts.api.model.Persona;
-import org.hawkular.accounts.api.model.Resource;
-import org.hawkular.accounts.sample.control.HawkularAccountsSample;
-import org.hawkular.accounts.sample.entity.Sample;
-import org.hawkular.accounts.sample.entity.SampleRequest;
-import org.hawkular.accounts.sample.entity.Sample_;
```

Comment:
By the way: you can build it locally by using the "sample" and "distribution" profiles: mvn clean install -Psample,distribution .